pkg/tsdb/interval/interval_test.go

```diff
@@ -70,6 +71,8 @@ func TestFormatDuration(t *testing.T) {
 }
 
 func TestGetIntervalFrom(t *testing.T) {
+	dsJson, err := simplejson.NewJson([]byte(`{"timeInterval": "60s"}`))
```

Comment:
Nit: But Go initialisms in consistent case etc:
```suggestion
	dsJSON, err := simplejson.NewJson([]byte(`{"timeInterval": "60s"}`))
```